src/patches/parts/hotkeys.cc

```diff
@@ -150,105 +120,106 @@ void ScreenManager_Update_Hook(auto original, ScreenManager* _this)
         } else {
           fleet_bar->RequestSelect(ship_select_request);
         }
+        return;
       }
     }
   }
 
-  if (GetKeyDownInt(KeyCode::Escape)) {
+  if (Key::Pressed(KeyCode::Escape) && (Key::IsInputFocused || Hub::IsInChat)) {
```

Comment:
I am going to assume you intended to call the functions here instead of having a "always evaluates to true" as you are now checking the pointers of the function

Reply:
Fixed, thanks.